docs/upgrade.html

```diff
@@ -175,6 +175,9 @@ <h5><a id="upgrade_400_notable" href="#upgrade_400_notable">Notable changes in 4
                         </li>
                         <li>The deprecated <code>sendOffsetsToTransaction(Map&lt;TopicPartition, OffsetAndMetadata&gt;, String)</code> method has been removed from the Producer API.
                         </li>
+                        <li>The default value of <code>linger.ms</code> has been changed from 0ms to 5ms. The impact of this is larger request batches will be filled by default at the expense of
+                            time in the producer. It is generally observed that values lower than 5ms do not yield any latency benefits due to the IO overhead of smaller requests.
```

Comment:
I would probably use similar wording to the one I suggested in a different comment.

Reply:
Done